### ISSUE TITLE
Run Commands On First Join

### ISSUE THREAD

**Issue by Kizwiz** [ProjectEssentials/ProjectEssentials#11]
When a player joins a server for the first time the server will run command(s) BUT the commands do not run if its returning players.

For example when a new player joins the server they are given a written book via command. But if they log out and log back in later they will not receive the book again as they are a returning player.

Asked on the CurseForge page, but posting here as suggested.

Thanks.

**Comment by MairwunNx** [comment]
### 👍 Thanks, @Kizwiz, good idea. I will start working on it. In the next minor version I will implement this feature.

Probably it will be placed in new configuration with name `general.json`, and property will have name `joinCommands`.

:D **Preview** usage example: 

`<>` - command arguments, separated via `,`.

```json
"jsonCommands": [
    "help",
    "ban <somebody once>",
    "mute <told me>, <the world is gonna roll me>"
]
```

**Comment by MairwunNx** [comment]
This configuration placed in `general.properties` in property `first-join-commands`, accepted data type is list, for example

```properties
first-join-commands=["/help", "/tpa MairwunNx"]
```

It also can be configured via command `/configure-essentials first-join-command set <your list value>`

Changed will included in `2.0.0-RC.1` version for both minecraft versions.
At now changed still in local git and no pushed to origin.

This issue will automatically closed when changes will pushed to origin.
